message/issue/experimental.md

```diff
@@ -0,0 +1,5 @@
+Hey @${author}! You seem to be running an experimental version (${version}) of ${repository_name} that should not be run on production systems.
+
+Plugin compatibility WILL break often on experimental systems, so unless you are a plugin developer assisting the next version of the API, please retest on API (${stable}).
```

Comment:
`WILL` -> `**will**`

Reply:
`please retest on API (${stable})` sounds weird.

Reply:
```Please re-test on the reccomended build or latest build of Sponge (${stable}). ``` ?

(credit to aaron)

Reply:
`retest` -> `reproduce`